docs/api.rst

```diff
@@ -43,6 +43,10 @@ iPython AutoReload
 .. automodule:: plum.autoreload
     :members:
 
+Overload Support
+----------------
+.. autofunction:: plum.overload.dispatch
```

Comment:
are you sure this function still exists?

Reply:
Should be gone now. :) Ty!